presto-docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -268,6 +271,40 @@ and a file system location of ``s3://test_bucket/test_schema/test_table``:
         location = 's3://test_bucket/test_schema/test_table')
     )
 
+Caching Support
+----------------
+
+Manifest File Caching
+^^^^^^^^^^^^^^^^^^^^^^
+
+Starting from Iceberg version 1.1.0, Apache Iceberg provides a mechanism to cache the contents of Iceberg manifest files in memory. This manifest caching feature helps
+to reduce repeated reads of small Iceberg manifest files from remote storage.
+
+.. note::
+
+    Currently, manifest file caching is supported for Hadoop and Nessie catalogs in the Presto Iceberg connector.
+
+The following configuration properties are available:
+
+====================================================   =============================================================   ============
+Property Name                                          Description                                                     Default
+====================================================   =============================================================   ============
+``iceberg.io.manifest.cache-enabled``                  Enable/disable the manifest caching feature. This feature       ``false``
```

Comment:
```suggestion
``iceberg.io.manifest.cache-enabled``                  Enable or disable the manifest caching feature. This feature       ``false``
```
Nit suggestion rephrase. See GitLab documentation style guide [recommended word list entry for slashes](https://docs.gitlab.com/ee/development/documentation/styleguide/word_list.html#slashes).